bin/install_openstackbmc.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -x
+
+yum -y update centos-release # required for rdo-release install to work
+yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+yum install -y https://rdo.fedorapeople.org/rdo-release.rpm
+yum install -y python-pip python-crypto os-net-config python-novaclient python-neutronclient git jq
+pip install --no-deps git+https://github.com/steveb/pyghmi.git@custom-bind-address
+
+cat <<EOF >/usr/local/bin/openstackbmc
+$openstackbmc_script
+EOF
+chmod +x /usr/local/bin/openstackbmc
+
+mkdir /etc/os-net-config
+echo "network_config:" > /etc/os-net-config/config.yaml
+
+for i in $(seq 2 $bm_node_count)
+do
+    nic="eth$(($i-1))"
+    echo "- {name: $nic, type: interface, use_dhcp: true, defroute: no}" >> /etc/os-net-config/config.yaml
+done
+os-net-config --verbose
+
+export OS_USERNAME=$os_user
+export OS_TENANT_NAME=$os_tenant
+export OS_PASSWORD=$os_password
+export OS_AUTH_URL=$os_auth_url
+
+for i in $(seq 1 $bm_node_count)
+do
+    bm_port="$bm_prefix_$(($i-1))"
```

Comment:
It might be nice to specify the variable as ${bm_prefix} so there's no confusion about what is the variable and what is literal text.